src/services/googleapi/googleapi.js

```diff
@@ -0,0 +1,47 @@
+let config = require('config.js');
+
+function queryGoogleApiData(resolve, reject, queryObj) {
+  let google = require('googleapis');
+  let analytics = google.analytics('v3');
+
+  let key = require('My Project-8172a1e558d9.json');
+  let VIEW_ID = config.googleAnalyticsViewId;
+
+  let jwtClient = new google.auth.JWT(
+    key.client_email,
+    null,
+    key.private_key,
```

Comment:
client_email and private_key can be fields in config.js.

They can be namespaced under something like `ga`

```
module.exports = {

  ...

  ga: {
    client_email: process.env.GA_CLIENT_EMAIL,
    private_key: process.env.GA_PRIVATE_KEY
  }

}

```